conntrack_linux.go

```diff
@@ -220,10 +223,11 @@ type ProtoInfo interface {
 type ProtoInfoTCP struct {
 	State uint8
 }
+
 // Protocol returns "tcp".
-func (*ProtoInfoTCP) Protocol() string {return "tcp"}
+func (*ProtoInfoTCP) Protocol() string { return "tcp" }
 func (p *ProtoInfoTCP) toNlData() ([]*nl.RtAttr, error) {
-	ctProtoInfo := nl.NewRtAttr(unix.NLA_F_NESTED | nl.CTA_PROTOINFO, []byte{})
+	ctProtoInfo := nl.NewRtAttr(unix.NLA_F_NESTED|nl.CTA_PROTOINFO, []byte{})
```

Comment:
Looks like some unrelated formatting changes made it in. It's a bit of a pain (my IDE also formats these on save), but better to handle those separate from this change;

- can you revert the formatting changes
- squash the commits? (second commit is fixing up the first one, and as the first one isn't merged yet, we may as well keep it as a single change).



I have a PR pending to fix formatting for the whole repo that's still pending; https://github.com/vishvananda/netlink/pull/1009

Reply:
Yeah, sorry. I should have checked that before pushing. I'll fix it now.

Reply:
No worries! It's a bit of a pain for sure!

I'm not a maintainer on this repository, but hope that "gofumpt" PR gets accepted and merged, as I find  myself undoing  formatting changes every time 😂